src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
+  const result = days * dayCost;
+  const shortTerm = 3;
+  const shortTermDisc = 20;
+  const longTerm = 7;
+  const longTermDisc = 50;
```

Comment:
```suggestion
  const DAY_PRICE = 40;
  const result = days * dayCost;
  const SHORT_TERM = 3;
  const SMALL_DISCOUNT = 20;
  const LONG_TERM = 7;
  const BIG_DISCOUNT = 50;
```
Consider following naming conventions